tests/organizations/test_get_program.py

```diff
@@ -0,0 +1,304 @@
+        test_user1.available_to_mentor = user1["available_to_mentor"]
+
+        test_user1.save_to_db()
+        self.test_user1_data = UserModel.find_by_email(test_user1.email)
+        AUTH_COOKIE["user"] = marshal(self.test_user1_data, full_user_api_model)
+
+        test_user_extension = UserExtensionModel(
+            user_id=self.test_user1_data.id,
+            timezone="AUSTRALIA_MELBOURNE"
+        )
+        test_user_extension.is_organization_rep = True
+        test_user_extension.save_to_db()
+
+        # prepare existing organization
+        # created on the 2020-09-30 10:00 AEST+1000
+        organization = OrganizationModel(
+            rep_id=self.test_user1_data.id, 
+            name="Company ABC",
+            email="companyabc@mail.com",
+            address="506 Elizabeth St, Melbourne VIC 3000, Australia",
+            website="https://www.ames.net.au",
+            timezone="AUSTRALIA_MELBOURNE",
+        )
+        organization.rep_department = "H&R Department"
+        organization.about = "This is about ABC"
+        organization.phone = "321-456-789"
+        organization.status = "DRAFT"
+        organization.join_date = 1601424000
+
+        organization.save_to_db()
+        self.organization1_data = OrganizationModel.find_by_representative(self.test_user1_data.id)
+        
+        # prepare expected representative object
+        self.expected_representative = marshal(self.test_user1_data, public_user_personal_details_response_model)
+
+        
+    @patch("requests.get")
+    def test_api_dao_get_program_successfully(self, mock_get_representative):
+        # prepare existing program with 
+        # start date 2020-10-15 12:00 GMT+0
+        # and end date 2020-11-25 12:00 GMT+0
+        program = ProgramModel(
+            program_name="Program A",
+            start_date=1602763200,
+            end_date=1606305600,
+            organization_id=self.organization1_data,
+        )
+        program.description = "This is about Program A."
+        program.target_skills = ["Python", "Ruby", "React"]
+        program.target_candidate = {
+            "target_candidate_gender": "Other",
+            "target_candidate_age": "Not Applicable",
+            "target_candidate_ethnicity": "Not Applicable",
+            "target_candidate_sexual_orientation": "Not Applicable",
+            "target_candidate_religion": "Not Applicable",
+            "target_candidate_physical_ability": "Not Applicable",
+            "target_candidate_mental_ability": "Not Applicable",
+            "target_candidate_socio_economic": "Not Applicable",
+            "target_candidate_highest_education": "Not Applicable",
+            "target_candidate_years_of_experience": "Not Applicable",
+            "target_candidate_other": "Non-Binary",
+        }
+        program.contact_type = "REMOTE"
+        program.zone = "GLOBAL"
+        program.status = "DRAFT"
+        program.creation_date = 1601424000
```

Comment:
can you use a code generated timestamp here please

Reply:
Done 😉